specification/window.browser.html

```diff
@@ -0,0 +1,1673 @@
+      <li><a href="#index-defined-elsewhere"><span class="secno"></span> <span class="content">Terms defined by reference</span></a>
+     </ol>
+    <li>
+     <a href="#references"><span class="secno"></span> <span class="content">References</span></a>
+     <ol class="toc">
+      <li><a href="#normative"><span class="secno"></span> <span class="content">Normative References</span></a>
+     </ol>
+    <li><a href="#idl-index"><span class="secno"></span> <span class="content">IDL Index</span></a>
+   </ol>
+  </nav>
+  <main>
+   <section class="non-normative">
+    <h3 class="heading settled" id="introduction"><span class="content">Introduction</span></h3>
+     <em>This section is non-normative.</em> 
+    <p>Modern WebExtensions are an evolution of a system originally introduced in
+  Chrome version 3. In that system, any API that was added was put under the
+ `chrome` namespace. When WebExtensions were adopted outside of Chromium, other
+  vendors used the more neutral `browser` namespace. That `browser` is now
+  considered to canonically correct namespace for all WebExtension APIs.
+  Some interactions with WebExtensions (such as external messaging from content
+  scripts to background processes) necessitate the exposure of this namespace
+  to the web.
+  Given that, this document defines `window.browser` as reserved for the use of
+  APIs designed to interact with WebExtensions. The specifics of what
+  APIs within `window.browser` are at present intentionally undefined, and are
+  left up to User Agent implementation.</p>
+   </section>
+   <section>
+    <h3 class="heading settled" id="window-interface"><span class="content"> <a class="idl-code" data-link-type="attribute" href="https://compat.spec.whatwg.org/#dom-window-orientation" id="ref-for-dom-window-orientation"><code>window.browser</code></a> API </span></h3>
+    <p><code class="idl"><a data-link-type="idl" href="#dom-window-browser" id="ref-for-dom-window-browser">browser</a></code> is UA defined attribute exposed on <code class="idl"><a data-link-type="idl" href="https://fetch.spec.whatwg.org/#dom-requestinit-window" id="ref-for-dom-requestinit-window">window</a></code>. When implemented,
+  it MUST be used only for WebExtension related functionality.</p>
+   </section>
+<pre class="idl highlight def"><c- b>partial</c-> <c- b>interface</c-> <a class="idl-code" data-link-type="interface" href="https://html.spec.whatwg.org/multipage/nav-history-apis.html#window" id="ref-for-window"><c- g>Window</c-></a> {
+  <c- b>readonly</c-> <c- b>attribute</c-> <a class="idl-code" data-link-type="interface" href="https://webidl.spec.whatwg.org/#idl-short" id="ref-for-idl-short"><c- b>short</c-></a> <dfn class="dfn-paneled idl-code" data-dfn-for="Window" data-dfn-type="attribute" data-export data-readonly data-type="short" id="dom-window-browser"><code><c- g>browser</c-></code></dfn>;
```

Comment:
`short` is the wrong type for this. `object` would be the best type to use here until we have a `Namespace` object type spec.

Reply:
updated